packages/contract-method-name/eth-name.js

```diff
@@ -0,0 +1,59 @@
+import {MethodRegistry} from 'eth-method-registry'
+import {fetchWithStorage} from '@cfxjs/utils'
```

Comment:
there's no need to introduce cache here, it's not the concern of this package

Reply:
The purpose of adding cache is to improve user experience. Because we  sending a  third part api here that we can not control and don not get clear of  there abilities of load balance

Reply:
Yeah, but it's not in the scope of this package.
It's better to split these functions so that user can decide where and if it's necessary to implement the cache.

Caching are often related to data model of user's product. For example, user may have multiple contract address modeled as dapp and simply cache on a dappId instead of contract address.

Reply:
ok